desktop/apps/consignments2/stylesheets/landing/desktop.styl

```diff
@@ -0,0 +1,172 @@
+
+  &__consign-button
+    background-color white
+    color black
+    &:hover
+      color purple-color
+      background-color white
+
+  &__headline
+    garamond-size('xl-headline')
+  &__subheadline
+    garamond-size('s-headline')
+
+  &__cta
+    margin gutter 0
+    > h3
+      margin (gutter / 2) 0
+      avant-garde-size('m-headline')
+    input
+      background-color white
+
+.consignments2-sections
+  // `.responsive-layout-container`-ish
+  margin 0 auto
+  padding 0 55px
+  @media screen and (min-width: 1440px)
+    padding 0 150px
+  @media screen and (min-width: 1660px)
+    padding 0 200px
+
+.consignments2-section
+  display flex
+  align-items center
+  padding gutter
+  border-bottom 1px solid gray-lighter-color
+
+  &__about
+    width 50%
+    padding gutter 0
+    &:first-child
+      padding-right (gutter * 4)
+    &:last-child
+      padding-left (gutter * 4)
+
+    &__headline
+      garamond-size('l-headline')
+    &__description
+      block-margins(gutter)
+      garamond-size('l-body')
+
+  &__images
+    width 50%
+    &:last-child
+      text-align right
+    > img
+      display inline-block
+      max-width 100%
+      vertical-align bottom
+
+    &.consignments2-section__images--logos
+      padding gutter
+    &__logo
+      width calc('50% - %s' % gutter)
+      display inline-block
+      margin (gutter / 2)
+      vertical-align middle !important
+
+.consignments2-steps
+  padding gutter
+  border-bottom 1px solid gray-lighter-color
+
+  &__headline
+    garamond-size('l-headline')
+
+  &__steps
+    display flex
+    justify-content space-between
+    margin gutter 0
+    &__step
+      width 32%
+      &__image
+        max-width 100%
+      &__description
+        margin 20px 0 40px
+        > span
+          float left
+          margin-right gutter
+          garamond-size('xxl-headline')
+        > p
+          padding 0 (gutter * 2)
+          garamond-size('l-body')
+
+.consignments2-recently-sold
+.consignments2-in-demand
+  padding gutter
+  border-bottom 1px solid gray-lighter-color
+
+.consignments2-recently-sold
+.consignments2-in-demand
+.consignments2-footer
+  h3
+    garamond-size('l-headline')
+
+.consignments2-in-demand
+  &__artists
+    display flex
+    justify-content space-between
+    flex-wrap wrap
+    margin gutter 0
+    &__artist
+      width 15%
+      margin-bottom gutter
+      &__thumbnail
+        display block
+        > img
+          width 100%
+      &__name
+        margin 10px 0
+        avant-garde-size('body')
+      &__button
+        //
+
+.consignments2-footer
+  text-align center
+  width 40%
+  margin 0 auto
+  padding (gutter * 2) 0 (gutter * 4)
+  > h3
+    margin-bottom gutter
+
+.consignments2-credit
+  garamond-size('s-body')
+  color gray-color
+  font-style italic
+
+
+mpv-height = 665px
+mpv-padding = 20px
+// narrow mpv height based on 1024 x 768 viewport
+// to catch some differing FAQ lengths
+mpv-height-narrow = 710px
+
+.faq-multi-page
```

Comment:
This might be copy + pasted from consignments 1, but make sure to namespace this by `consignments2` otherwise it will conflict with consignments 1 if you iterate on it.